libraries/AP_Mount/AP_Mount_Backend.h

```diff
@@ -212,6 +212,8 @@ class AP_Mount_Backend
     // enable/disable rangefinder.  Returns true on success
     virtual bool set_rangefinder_enable(bool enable) { return false; }
 
+    void vehicle_has_started_to_land();
```

Comment:
Thanks for this.  Let's put a comment above this method to clarify that this should be called from the vehicle cold when the vehicle has started landing

Reply:
Done